cli/src/commands/pull.ts

```diff
@@ -1,3 +1,4 @@
+import { throwError as observableThrowError, Observable } from 'rxjs';
```

Comment:
Is this alias also by `tslint-rxjs`? I think I like `throwError` more as it's in line with `catchError`.

Reply:
This was very likely done by rxjs-tslint yes. Will fix this.